qiskit/extensions/standard/u3.py

```diff
@@ -31,15 +29,6 @@ def __init__(self, theta, phi, lam, label=None):
         """Create new two-pulse single qubit gate."""
         super().__init__("u3", 1, [theta, phi, lam], label=label)
 
-    def _define(self):
```

Comment:
I see `_define` used once in the codebase:
```
qiskit-terra/qiskit/circuit/instruction.py:
  158          """Return definition in terms of other basic gates."""
  159          if self._definition is None:
  160:             self._define()
  161          return self._definition
```
Could the `U3Gate` reach this? Where is `self._definition` set for this gate, perhaps through inheritance (or I just missed it)? Same questions for `CnotGate`

Reply:
Yes... it will be executed by inheritance. As before.